bin/gremlin.sh

```diff
@@ -1,22 +0,0 @@
-#!/bin/bash
```

Comment:
The diff makes it look like the file was removed and then added back? Either way, any reason we don't have the apache license header here?